source/engine/graphics/resources/CubeMapManager.h

```diff
@@ -1,8 +1,8 @@
 #pragma once
```

Comment:
Is this file accidentally hanging around? There is also `cubemaps/CubeMapManager.h`.